outputs.tf

```diff
@@ -1,23 +1,45 @@
 output "fargate_service" {
-  value = module.fargate.ecs_service
+  value = aws_ecs_service.service
 }
 
 output "fargate_service_security_group" {
-  value = module.fargate.service_sg
+  value = aws_security_group.fargate_service_sg
 }
 
-output "codedeploy_appspec_json" {
-  value = module.fargate.codedeploy_appspec_json
+output "task_definition" {
+  value = aws_ecs_task_definition.task_def
+}
+
+output "codedeploy_deployment_group" {
+  value = aws_codedeploy_deployment_group.deploymentgroup
+}
+
+output "codedeploy_basic_appspec_json" {
```

Comment:
What would you think if we made the fargate module create an appspec file instead of give an output? It is just a thought to make it easier to run pipelines.

Reply:
Done. For some reason trying to use yaml didn't work. AWS's codedeploy yaml parser is a bit odd, and I didn't want to dive into why it didn't work.